apps/estuary/lib/estuary_web/live/event_live_view.ex

```diff
@@ -56,4 +57,19 @@ defmodule EstuaryWeb.EventLiveView do
 
     {:noreply, push_patch(socket, to: Routes.live_path(socket, __MODULE__, search_params))}
   end
+
+  defp all_events(socket_events, filter_param) do
+    cond do
+      filter_param == nil ->
+        {:ok, events} = EventRetrievalService.get_all()
+        events
+
+      filter_param == "" ->
+        {:ok, events} = EventRetrievalService.get_all()
+        events
+
+      true ->
+        socket_events
+    end
```

Comment:
I'm a little confused at to what's happening here. There is no clear way to get to the `true` branch, and the other two are identical. Maybe clean it up with an `or` and put the check in the `cond`?